frontend/src/component/admin/apiToken/ApiTokenForm/ApiTokenForm.tsx

```diff
@@ -95,23 +94,29 @@ const ApiTokenForm: React.FC<IApiTokenFormProps> = ({
     handleCancel,
     errors,
     clearErrors,
+    scope = 'global',
 }) => {
     const { uiConfig } = useUiConfig();
     const { environments } = useEnvironments();
     const { projects: availableProjects } = useProjects();
 
+    const disableProjectSelection = scope === 'project';
+
```

Comment:
Wondering if we should change the architecture so these can be composed together instead of injecting internal state into one component.

Something like:

```
<ApiTokenForm>
   <ProjectSelector />
</ApiTokenForm>
```